nss-cohort-36/keahua-arboretum-net-surfers
# Initial plant files

# Description

Initial creation of plants files.

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout initial-plant-files`
4. Make sure all plant files are in plants dir as well as add_plant.py in action dir.

# Checklist:
